spark-commons/src/main/scala/za/co/absa/spark/commons/sql/functions.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spark.commons.sql
+
+import org.apache.spark.sql.Column
+import org.apache.spark.sql.functions.col
+import za.co.absa.spark.commons.utils.SchemaUtils
+
+import scala.util.{Success, Try}
+
+// scalastyle:off
+object functions {
```

Comment:
Both the object name and the function name are breaking (our) naming conventions. But they follow the naming of _Spark_, so for consistency I decided to follow.